Examples/Swift/MultipleImagesExample.swift

```diff
@@ -0,0 +1,49 @@
+
+import Mapbox
+
+@objc(MultipleImagesExample_Swift)
+
+class MultipleImagesExample: UIViewController, MGLMapViewDelegate {
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // Create and add a map view.
+        let mapView = MGLMapView(frame: view.bounds, styleURL: MGLStyle.outdoorsStyleURL)
+        
+        // Center the map on Yosemite National Park, United States.
+        mapView.setCenter(CLLocationCoordinate2D(latitude: 37.761, longitude: -119.624), zoomLevel: 10, animated: false)
+        mapView.autoresizingMask = [.flexibleHeight, .flexibleWidth]
+        mapView.delegate = self
+        view.addSubview(mapView)
+    }
+    
+    func mapView(_ mapView: MGLMapView, didFinishLoading style: MGLStyle) {
+        // Add icons from the U.S. National Parks Service to the map's style.
+        style.setImage(UIImage(named: "nps-restrooms")!, forName: "restrooms")
+        style.setImage(UIImage(named: "nps-trailhead")!, forName: "trailhead")
+        style.setImage(UIImage(named: "nps-picnic-area")!, forName: "picnic-area")
+        
+        // Access a vector tileset that contains places of interest at Yosemite National Park. This tileset was created by uploading NPS shapefiles to Mapbox Studio.
+        if let url = URL(string: "mapbox://examples.ciuz0vpc") {
```

Comment:
This is creating a URL from a string literal, so there’s no need to account for the case where the URL is malformed. `!` would be appropriate here.